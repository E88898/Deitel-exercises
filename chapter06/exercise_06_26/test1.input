6 16 53
6 17 53
