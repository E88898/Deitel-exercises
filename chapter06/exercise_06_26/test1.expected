Amount of time in seconds between two time is: 60 seconds
